### PR TITLE
Charts plugin development tutorial

### PR DESCRIPTION
**Pull request by shiltemann**
tutorial and slides for @guerler and my workshop for GCC

### CODE REVIEW

**Comment by bgruening** [comment]
@guerler @shiltemann this is a very beautiful one! Thanks a lot!

File: topics/dev/tutorials/interactive-environments/slides.html [hunk 1]
```diff
@@ -30,10 +34,38 @@
...
+![](../../images/which_viz_flowchart.png)
+
+
+---
+
 ### How to launch an IE?
 
 - Can be bound to specific datatypes
-  - Available under the visualizations button on the dataset
+  - Available under the       zations button on the dataset
```
Comment (bgruening) [inline topics/dev/tutorials/interactive-environments/slides.html]: this looks wrong ...

Comment (shiltemann) [inline topics/dev/tutorials/interactive-environments/slides.html]: huh? oops!

**Comment by shiltemann** [comment]
@guerler, please check that you agree with these changes. And should we put the video on youtube/vimeo?

**Comment by bgruening** [comment]
I vote for Vimeo/Youtube :)
Thanks @shiltemann looks good to me.

**Comment by martenson** [comment]
@shiltemann vimeo, it has no adds and we have premium account, if you share it with me I can upload under galaxyproject

**Comment by guerler** [comment]
Yeah its in my google drive. https://drive.google.com/drive/folders/0ByOGvyChhu5dSkVmdDUyS1Vfc0U

**Comment by martenson** [comment]
@guerler https://vimeo.com/222709200

**Comment by bgruening** [comment]
Thanks @shiltemann @martenson @guerler!

**Comment by guerler** [comment]
Thanks everyone.

**Comment by bgruening** [comment]
@guerler ready for merge?

**Comment by guerler** [comment]
Sure. Looks good to me. Thanks a lot for you help @shiltemann.

**Pull request merged**